op-chain-ops/deployer/pipeline/opchain.go

```diff
@@ -62,7 +62,7 @@ func DeployOPChain(ctx context.Context, env *Env, intent *state.Intent, st *stat
 						BasefeeScalar:          1368,
 						BlobBaseFeeScalar:      801949,
 						L2ChainId:              chainID.Big(),
-						Opsm:                   st.ImplementationsDeployment.OpsmAddress,
```

Comment:
Who deploys the OPSM implementation now?

Reply:
The implementation and proxy are both deployed within `DeployImplementations`